modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc

```diff
@@ -0,0 +1,78 @@
+
+NOTE: The *Create Workspace* page of your {prod-short} dashboard has the *Git Repo URL ** field as the limited-functionality GUI alternative for entering the URL of a Git repository to start a new workspace.
+
+.Prerequisites
+
+* Your organization has a running instance of {prod-short}.
+* The administrator of your organization's {prod-short} instance or the repository maintainer or the README author has provided you with the URL of your organization's {prod-short} instance that you need as a user to access your {prod-short} dashboard pages and start and manage your workspaces. This URL is a fully qualified domain name (FQDN): `pass:c,a,q[{prod-url}]`.
+* Your Git repository maintainer keeps the `devfile.yaml` or `.devfile.yaml` file in the root directory of the Git repository. (For alternative file names and file paths, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].)
++
+IMPORTANT: You can also start a new workspace by supplying the URL of a Git repository that contains no devfile. Doing so results in a workspace with the Che-Theia IDE and the Universal Developer Image.
+//provide xrefs to a concept module about the Universal Developer Image and, if applicable, devfile-less defaults for new workspaces. max-cx
+
+.Procedure
+
+. Optional: Visit your {prod-short} dashboard page to authenticate as a {prod-short} user to your organization's instance of {prod-short}.
+
+. Visit the URL to start a new workspace using the basic syntax:
+[source,subs="+quotes,+attributes,+macros"]
++
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__
+----
++
+[TIP]
+====
+You can extend this URL with optional parameters:
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
+----
+<1> See xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].
+====
++
+.URL syntax for starting a new workspace that contains a clone of a GitHub-hosted repository
+====
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/pull/__<pull_request_id>__]` starts a new workspace with a clone of the branch of the pull request.
+====
++
+After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page.
++
+When the new workspace is ready, the workspace GUI loads in the browser tab.
++
+A clone of the Git repository is present in the filesystem of the new workspace.
++
+The workspace has a unique URL: `pass:c,a,q[{prod-url}]#workspace__<unique_url>__`.
+
+[TIP]
```

Comment:
@rolfedh, this is how I rephrased the lead-in sentence in this admonition:
![image](https://user-images.githubusercontent.com/78564304/161952957-e8cae559-16a8-40eb-9a76-758875d7e040.png)